Modules/Devices/src/Devices.Domain/Entities/Identities/IdentityDeletionProcessAuditLogEntry.cs

```diff
@@ -28,6 +28,17 @@ private IdentityDeletionProcessAuditLogEntry()
         NewStatus = newStatus;
     }
 
+    private IdentityDeletionProcessAuditLogEntry(IdentityDeletionProcessId processId, string message, byte[] identityAddressHash, DeletionProcessStatus? oldStatus, DeletionProcessStatus newStatus)
```

Comment:
If you continue to use the constructor, you should use it in all places where it is applicable. E.g. `ProcessStartedBySupport`

Reply:
Changes made (constructor removed in previous step).